icd.c

```diff
@@ -198,6 +198,12 @@ void khrIcdVendorAdd(const char *libraryName)
 
 void khrIcdContextPropertiesGetPlatform(const cl_context_properties *properties, cl_platform_id *outPlatform)
 {
+    if (properties == NULL && khrIcdVendors != NULL)
```

Comment:
I'm fine solving the NULL properties case now and solving the non-NULL (but no CL_CONTEXT_PLATFORM) case later, if needed.